platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -27,44 +27,24 @@ For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. -->
 
-<!--
-==========================================================
-ADD RELEASE FEATURES BELOW
-vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
-
-<a name="dependency-resolution"></a>
-### Dependency management improvements
-
-Gradle's [dependency management](userguide/core_dependency_management.html) infrastructure allows projects to depend on artifacts from external repositories, other projects from the same build, or projects from included builds.
-
-#### Root variant exposed by `ResolutionResult`
-
-The [`ResolutionResult` API](javadoc/org/gradle/api/artifacts/result/ResolutionResult.html), which contains the results of dependency resolution, now exposes the root variant of the resolved graph in addition to its owning component. 
-
-Previously, the API only exposed the root component, which exposed the first-level dependencies as well dependencies from other selected variants in the root component:
-
-```groovy
-def rootComponent = configurations.runtimeClasspath.incoming.resolutionResult.rootComponent
-```
+<a name="native-toolchains"></a>
+### Native toolchains support
 
-In this release, the root variant is also exposed.
-The root variant is a synthetic variant representing the `Configuration` being resolved, and exposes the first-level dependencies of a resolution.
+Gradle's [native toolchain support](userguide/native_software.html#native_binaries:tool_chain) enables building C/C++ projects with compilers like GCC, Clang, and Visual C++.
 
-```groovy
-def rootVariant = configurations.runtimeClasspath.incoming.resolutionResult.rootVariant
-```
+#### Linux arm64 as a build target
 
-The updated API allows for more precise traversal of dependency graphs by operating at the variant level instead of just the component level. 
-By traversing dependencies at the variant level, it is possible to differentiate between the production code of a component and its test fixtures.
+This release adds Clang and GCC support to Gradle on Linux platforms that use the ARM or AArch64 architecture.
 
-For more details on how to perform a graph traversal, see [Programmatic Dependency Resolution](userguide/programmatic_dependency_resolution.html).
+Thank you to [Matthew Von-Maszewski](https://github.com/matthewvon) for the contribution.
```

Comment:
🤔 "Thank you to" reads strangely. Maybe:
> Thanks to

Reply:
This comes from @adammurdoch. I wasn't sure if we were in the habit of putting Thank you's outside the contributor section. Is this something we have done before?